CMakeLists.txt

```diff
@@ -320,7 +333,7 @@ if(UNIX)
 elseif(WIN32)
     get_filename_component(LIBTUNTAP_IMPL ${TT_ROOT}/tuntap-windows.c ABSOLUTE)
     get_filename_component(EV_SRC "llarp/ev/ev_win32.cpp" ABSOLUTE)
-    add_definitions(-DWIN32_LEAN_AND_MEAN -DWIN32 -DWINVER=0x500 -D_WIN32_WINNT=0x500)
+    add_definitions(-DWIN32_LEAN_AND_MEAN -DWIN32 -DWINVER=0x0602 -D_WIN32_WINNT=0x0602)
```

Comment:
set Windows 8 requirement conditionally, the UNIX tools are slightly dated because they track reactos

Reply:
👍 I'm just trying to get to a stage where this builds + tests pass first 😉

Reply:
I

Reply:
ah I missed this.